src/ISDisplay.h

```diff
@@ -138,6 +139,8 @@ class cInertialSenseDisplay
 	bool UploadNeeded() { bool uploadNeeded = m_editData.uploadNeeded; m_editData.uploadNeeded = false; return uploadNeeded; };
 	edit_data_t *EditData() { return &m_editData; }
 	void setOutputOnceDid(int did) { m_outputOnceDid = did; m_interactiveMode = m_outputOnceDid == 0; }
+	void SetSerialPort(serial_port_t* port) { m_port = port; }
```

Comment:
How does this work when connected to multiple devices (ie `-c *`)?  How does normal/current display stuff work in that situation?

Reply:
It only shows stats for the first device.